src/main/assembly/dist/md/ddm/ddm.xsd

```diff
@@ -191,10 +193,10 @@
     </xs:element>
 
     <!-- =================================================================================== -->
-    <xs:element name="accessRights" substitutionGroup="dcterms:accessRights" type="ddm:EasyAccessRightsType">
+    <xs:element name="accessRights" type="ddm:EasyAccessRightsType">
```

Comment:
I assume that, removing the `substitutionGroup` attribute does mean that it is no longer possible to use the `ddm:accessRights` element where `dcterms:accessRights` is required. Not sure if that is ever going to be problem.